src/observers/TelemetryObserver.ts

```diff
@@ -47,9 +47,9 @@ export class TelemetryObserver {
             case EventType.ProjectConfigurationReceived:
                 let projectConfig = (<ProjectConfiguration>event).projectConfiguration;
                 telemetryProps['ProjectGuid'] = projectConfig.ProjectGuid;
```

Comment:
I believe you should also change this to `ProjectId`. Right @DamianEdwards?

Reply:
Done. Thanks!

Reply:
Perfect :-)